README.md

```diff
@@ -2,11 +2,35 @@
 
 A Survey of:
 
-_Does the first programming language someone learns influence whether they prefer R or Python (or have no preference)?_
+_Does the first programming language learned influence whether someone prefersPython or R when it comes to data analysis._
```

Comment:
_Does the first programming language learned influence whether someone prefers Python or R when it comes to data analysis?_